free-1.md

```diff
@@ -58,11 +117,31 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I don't think that there would be many systems aspects of the language, as the
+overall concept of crochet pattern creations is creative, there would not be
+too many limitations. I think there would be lots of space to work on the design
+both in the language creation and in the two output options.
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+I think it depends on how much work is put into the interactive aspect of the
+project. I imagine that someone could definitely create a very clean language
+and the PDF output. There would then be room for the person to add the
+interactive output, and maybe add more ways for the user to be creative in their
+pattern creation (like adding new stitch definitions).
```

Comment:
I think this is a very flexible project in that there are always features that you can add or subtract in order to make it better fit the scope. The basic idea of having a DSL that manages patterns seems quite doable.